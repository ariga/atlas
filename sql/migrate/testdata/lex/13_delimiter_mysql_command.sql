# Test delimiter :
select "Test delimiter :" as " ";
delimiter :
select * from t1:
/* Delimiter commands can have comments */
delimiter ;
select 'End test :';

/* Test delimiter :; */
select "Test delimiter :;" as " ";
delimiter :;
select * from t1 :;
delimiter ;
select 'End test :;';

-- Test delimiter //
select "Test delimiter //" as " ";
delimiter //
select * from t1//
delimiter ;
select 'End test //';

# Test delimiter 'MySQL'
select "Test delimiter MySQL" as " ";
delimiter 'MySQL'
select * from t1MySQL
delimiter ;
select 'End test MySQL';

# Test delimiter 'delimiter'
select "Test delimiter delimiter" as " ";
delimiter delimiter
select * from t1delimiter
delimiter ;
select 'End test delimiter';

# Test delimiter @@
select "Test delimiter @@" as " ";
delimiter @@
select * from t1 @@
select * from t2@@
alter table t add column c@@
delimiter ;
select 'End test @@';

# Test delimiter \n\n
select "Test delimiter \n\n" as " ";
delimiter \n\n
select * from t1

select * from t2

delimiter ;
select 'End test \\n\\n';