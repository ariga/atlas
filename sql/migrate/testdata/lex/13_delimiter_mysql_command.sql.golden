select "Test delimiter :" as " ";
-- end --
select * from t1
-- end --
select 'End test :';
-- end --
select "Test delimiter :;" as " ";
-- end --
select * from t1
-- end --
select 'End test :;';
-- end --
select "Test delimiter //" as " ";
-- end --
select * from t1
-- end --
select 'End test //';
-- end --
select "Test delimiter MySQL" as " ";
-- end --
select * from t1
-- end --
select 'End test MySQL';
-- end --
select "Test delimiter delimiter" as " ";
-- end --
select * from t1
-- end --
select 'End test delimiter';
-- end --
select "Test delimiter @@" as " ";
-- end --
select * from t1
-- end --
select * from t2
-- end --
alter table t add column c
-- end --
select 'End test @@';
-- end --
select "Test delimiter \n\n" as " ";
-- end --
select * from t1
-- end --
select * from t2
-- end --
select 'End test \\n\\n';